server.md

```diff
@@ -17,13 +17,13 @@ Each Edge server should come with a matching client library, typically named "ed
 
 For instance, the client library "edge-sync-client" would include the logic needed to maintain a repository on disk, identifying out-of-date files and synchronizing changes with the server. The client library "edge-info-client" might know how to fetch lists of resources from the info server, as well as logic for perfoming client-side load balancing using these resource lists.
 
-These client libraries are versioned independently from the server they communicate with, since they can implement bug fixes and add features without necessarily requiring server changes (and vice-versa).
+These client libraries are versioned independently from the server they communicate with, since they can implement bug fixes and add features without necessarily requiring server changes (and vice-versa). Client libraries should be versioned using [Samantic Versioning](https://semver.org/).
 
-Servers are encouraged to share code with their client libraries, such as type definitions, cleaners, constant definitions, and so forth. Putting the shared code in the client library is the most convenient option, since the client would be published to NPM for easy consumption by the server. However it is also possible to share code the other way around (see below).
+Servers are encouraged to share code with their client libraries, such as type definitions, cleaners, constant definitions, and so forth. Putting the shared code in the client library is the most convenient option, since the client would be published to NPM for easy consumption by the server. However it is also possible to share code the other way around; from the server (see below).
```

Comment:
The "; from the server (see below)" part is weird grammar. Maybe "(from the server, as below)." would be better.

Reply:
I went ahead and fixed the weird grammar here by using your suggestion.